src/main/java/edu/cornell/kfs/fp/batch/xml/AccountingXmlDocumentAccountingLine.java

```diff
@@ -54,6 +56,58 @@
     @XmlJavaTypeAdapter(KualiDecimalXmlAdapter.class)
     protected KualiDecimal amount;
 
+    @XmlElement(name = "base_amount", namespace = StringUtils.EMPTY, required = false)

```

Comment:
I'm wondering if we should add currentBudgetAdjustmentAmount?

Reply:
I was trying to reuse the existing POJO "amount" field for holding the value of the current BA amount field. And according to the code in BudgetAdjustmentDocument, it seems the regular amount is set to the value of the current BA amount when preparing to take action on the document.

If you think the naming conventions would be better if we had a "current_amount" field instead, please let me know.

Reply:
I don't have a strong opinion, but wanted to raise the concern.  Since BudgetAdjustmentDocument is reusing amount in the background, I am fine leaving this as is